### PR TITLE
Describe Overlay - hideOnParentScroll

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

**Review by alexanderPolosatov — APPROVED** [review]
LGTM

**Review by ksercs — APPROVED** [review]

File: api-reference/10 UI Components/dxPopover/1 Configuration/hideOnParentScroll.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ default: true
...
 ##### shortDescription
-<!-- Description goes here -->
+<!-- %shortDescription% -->
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+<!-- Description goes here -->
+
+<!-- import * from 'api-reference\10 UI Components\dxOverlay\1 Configuration\hideOnParentScroll.md' -->
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPopover/1 Configuration/hideOnParentScroll.md]: Popover inherits Overlay via Popup. You can safely remove the short and full decriptions - they will be inherited.

**Review by DokaRus — APPROVED** [review]

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Components/dxOverlay/1 Configuration/hideOnParentScroll.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to hide the {WidgetName} when users scroll one of the {WidgetName}'s parent elements.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxOverlay/1 Configuration/hideOnParentScroll.md]: Is it necessary to mention the WidgetName twice? For example, can we update this to the following?

`Specifies whether to hide the {WidgetName} when users scroll one of its parent elements.`

Comment (AlexJameson) [reply api-reference/10 UI Components/dxOverlay/1 Configuration/hideOnParentScroll.md]: Sounds reasonable, thanks

Comment (albertov05) [reply api-reference/10 UI Components/dxOverlay/1 Configuration/hideOnParentScroll.md]: Thank you

**Pull request merged**